docsrc/Connecting_and_queries.rst

```diff
@@ -626,3 +626,12 @@ You can import this module as follows:
 
     from datetime import datetime
 
+Execution timeout
+==============================
+
+The Firebolt Python SDK allows you to set a timeout for query execution.
+In order to do this, you can call the :meth:`Cursor.execute` or :meth:`Cursor.executemany` function with the
+``timeout_seconds`` parameter provided. In case the timeout will be reached before the query execution finishes, the
```

Comment:
Let's add an example, it's easier to understand where the parameter should be added.